automation/project/Settings.scala

```diff
@@ -23,17 +23,18 @@ object Settings {
     "-feature",
     "-encoding", "utf8",
     "-target:jvm-1.8",
-    "-Xmax-classfile-name", "100"
+    "-Xmax-classfile-name", "100",
+    "-Xfatal-warnings"
   )
 
   // test parameters explanation:
   // `-o` - causes test results to be written to the standard output
   //     `F` - Display full stack traces
   //     `D` - Display test duration after test name
-  //     `G` - show reminder of failed and canceled tests with full stack traces at the end of log file
+  //     (removed on April 22, 2018) `G` - show reminder of failed and canceled tests with full stack traces at the end of log file
```

Comment:
Why remove this?

Reply:
to reduce cluttering in log. In the same folder, `Test-Summary.log` provides same information.